### PR TITLE
FOUR-12575 Bold variables in Case Title Columns

### PR DESCRIPTION
**Pull request by caleeli**
## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-12575

## Code Review Checklist
- [x] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [x] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [x] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [x] This solution fixes the bug reported in the original ticket.
- [x] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [x] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [x] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [x] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [x] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:next
ci:deploy

### CODE REVIEW

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-987bc74145.eng.processmaker.net

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-987bc74145.eng.processmaker.net

File: database/migrations/2023_12_11_180034_add_case_title_formatted_to_process_requests_table.php
Comment (julceslauhub) [inline database/migrations/2023_12_11_180034_add_case_title_formatted_to_process_requests_table.php:1]: Change the table struct two times maybe can be affect the performance in migration. Process Request is a big table. Maybe be can only have one migration script to add two columns.

Comment (caleeli) [reply database/migrations/2023_12_11_180034_add_case_title_formatted_to_process_requests_table.php:1]: This will be handled in the ticket FOUR-12303

**Comment by tokensPM** [comment]
QA server was successfully deployed https://ci-987bc74145.eng.processmaker.net

**Review by devmiguelangel — APPROVED** [review]

**Pull request merged**